patches/chrome-browser-ui-views-page_info-page_info_bubble_view.cc.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc b/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
+index ebe5987d5b4239e09a9b255b0d73ec9f7ad74fe9..933b15b53b86376e63a425701ae8cb8571d8b692 100644
+--- a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
```

Comment:
I can't avoid this patch file because multiple `PageInfoBubbleView::CreatePageInfoBubble()` is used in this file and `InternalPageInfoBubbleView` class is in anonymous namespace.